external/buildscripts/build_unityscript_bareminimum_win.pl

```diff
@@ -65,19 +65,14 @@ sub UnityBooc
 
 sub BuildUnityScriptForUnity
 {	
-	# Build system is handling this
-	if (!$ENV{UNITY_THISISABUILDMACHINE}) {
-		GitClone("git://github.com/Unity-Technologies/boo.git", $booCheckout);
-	}
+	GitClone("git://github.com/Unity-Technologies/boo.git", $booCheckout);
```

Comment:
Do we always want to do this now?

Reply:
Its needed for OSX and bare minimum builds on Gitlab CI